packages/belos/src/BelosSolverManager.hpp

```diff
@@ -43,6 +43,90 @@
 #ifndef BELOS_SOLVERMANAGER_HPP
 #define BELOS_SOLVERMANAGER_HPP
 
+// These are here for convenience while developing the DII
+// I wanted to keep each cpp file as minimal as possible but not sure where
+// these macros could best live.
+
+// register the managers for a belos lib setup, such as the test Belos_Factory
+#define DEFINE_REGISTER_SOLVER_MANAGER(manager,name)              \
```

Comment:
Should namespace all these macros: `BELOS_DEFINE_REGISTER_SOLVER_MANAGER` etc.

Reply:
Some macros were eliminated with the single cpp format. I changes the remaining ones to be Belos namespaced.